common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -272,6 +271,8 @@ class EntitlementEnrollmentViewSetTest(ModuleStoreTestCase):
     def setUp(self):
         super(EntitlementEnrollmentViewSetTest, self).setUp()
         self.user = UserFactory()
+        UserFactory(username=settings.ECOMMERCE_SERVICE_WORKER_USERNAME, is_staff=True)
```

Comment:
Could you explain why we need this user created here if we don't use it anywhere else?

Reply:
Nevermind found it down lower, just not referencing the object directly and instead using the .get() call

Reply:
This is the Ecommerce service worker that is used in the tested code when calling into the Ecommerce service.